bluesky/run_engine.py

```diff
@@ -464,7 +464,8 @@ def setup_run_permit():
             'clear_checkpoint': self._clear_checkpoint,
             'rewindable': self._rewindable,
             'pause': self._pause,
-            'resume': self._resume,
+            '_resume_from_suspender': self._resume,
```

Comment:
I understand `self._resume` is triggered via a message from the suspender, but `self.resume` is called from an iPython terminal by the user. They seem to do very different things however, is there a case for making `self.resume` push a plan onto the stack in the same way as this PR does?

Would avoid this nastiness: https://github.com/coretl/bluesky/blob/2fce3f153073e73a916c92374d3452a62c22b426/bluesky/run_engine.py#L938-L939

Reply:
yes, I think there is a case for that.